client/src/ui/components/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import cx from 'classnames';
+
+interface ButtonProps {
+  children?: React.ReactNode;
+  iconStart?: React.ReactNode;
+  iconEnd?: React.ReactNode;
+  className?: string;
+  color: 'red' | 'grey';
+  disabled?: boolean;
+  size?: 'normal' | 'large';
+  isSubmit?: boolean;
+  onClick?(event?: React.FormEvent): void;
+}
+
+export const Button = ({ children, ...props }: ButtonProps) => {
+  const {
+    iconEnd,
+    iconStart,
+    color,
+    disabled,
+    className,
+    size,
+    isSubmit,
+    onClick,
+  } = props;
+  return (
+    <button
```

Comment:
we should pass the disabled prop too otherwise if disabled is true, you can still click on the button